pyteal/ast/acct.py

```diff
@@ -47,13 +90,97 @@ def authAddr(cls, acct: Expr) -> MaybeValue:
             acct: An index into Txn.accounts that corresponds to the application to check or an address available at runtime.
                 May evaluate to uint64 or an address.
         """
-        require_type(acct, TealType.anytype)
-        return MaybeValue(
-            Op.acct_params_get,
-            TealType.bytes,
-            immediate_args=["AcctAuthAddr"],
-            args=[acct],
-        )
+        return cls.__makeAccountParamExpr(AccountParamField.auth_addr, acct)
+
+    @classmethod
+    def totalNumUint(cls, acct: Expr) -> MaybeValue:
+        """Get the total number of uint64 values allocated by the account in Global and Local States.
```

Comment:
minor: it would be nice to mention in the docstrings for the new fields that they have a minimum AVM version of 8

Reply:
Yes, thank you for thinking of it!